### PR TITLE
correct data type

### PR DESCRIPTION
**Pull request by cybwan**
Avoid following code that will never be called.

### CODE REVIEW

File: kernel/loxilb_libdp.c [hunk 1]
```diff
@@ -1320,7 +1320,7 @@ llb_clear_map_stats(int tid, __u32 idx)
   t = &xh->maps[tid];
   if (t->has_pb && t->pb_xtid <= 0) {
     /* FIXME : Handle non-pcpu */
-    if (idx >= 0) {
+    if ((__s32)idx >= 0) {
```
Comment (UltraInstinct14) [inline kernel/loxilb_libdp.c:1323]: Typecasting to __s32 means anything greater than 2(pow)16 will fail this check which is not exactly desired. Suggest to revert this change in this file for now.

**Review by UltraInstinct14 — CHANGES_REQUESTED** [review]
Changes in natlbfwd.c looks good but kindly refer to comment for changes to libdp.c

File: kernel/loxilb_libdp.c (hunk 1 above)
Comment (TrekkieCoder) [reply kernel/loxilb_libdp.c]: This check is removed completely as it was unnecessary. Thanks @cybwan
